helm/mojaloop-barebone/config/application-fin12.properties

```diff
@@ -0,0 +1,2 @@
+ams.localenabled=true
+zeebe.broker.contactpoint=zeebe-zeebe-gateway:26500
```

Comment:
this name comes from the release name ( it may not be zeebe-zeebe)

Reply:
It is not used anywhere. Name is coming from release name only.